testing/conmon/APKBUILD

```diff
@@ -0,0 +1,25 @@
+# Contributor: Simon Ser <simon.ser@intel.com>
+# Maintainer: Simon Ser <simon.ser@intel.com>
+pkgname=conmon
+pkgver=2.0.0
+pkgrel=1
+pkgdesc="An OCI container runtime monitor"
+url="https://github.com/containers/conmon"
+arch="all"
+license="Apache-2.0"
+makedepends="glib-dev"
+depends="runc glib"
+source="$pkgname-$pkgver.tar.gz::https://github.com/containers/conmon/archive/v$pkgver.tar.gz"
+options="!check"
+
+build() {
+	cd "$srcdir/conmon-$pkgver"
```

Comment:
use builddir="$srcdir/conmon-$pkgver" in the root of the APKBUILD instead of cd'ing into it manually